rotkehlchen/inquirer.py

```diff
@@ -92,6 +95,7 @@
 log = RotkehlchenLogsAdapter(logger)
 
 CURRENT_PRICE_CACHE_SECS = 300  # 5 mins
+SATOSHI_PER_BSQ = FVal('0.00000100')
```

Comment:
very wrong name, can lead to misunderstanding in the future. 
```suggestion
BTC_PER_BSQ = FVal('0.00000100')
```